extensions/adobe/experience/journeyOrchestration/stepEvents/journeyStepEventCommonFieldsMixin.json

```diff
@@ -0,0 +1,172 @@
+              "type":"string"
+            },
+            "https://ns.adobe.com/experience/journeyOrchestration/stepName":{
+              "title":"stepName",
+              "description":"Name of the step that is currently in processing.",
+              "type":"string"
+            },
+            "https://ns.adobe.com/experience/journeyOrchestration/stepType":{
+              "title":"stepType",
+              "description":"Type of the step.",
+              "type":"string"
+            },
+            "https://ns.adobe.com/experience/journeyOrchestration/stepStatus":{
+              "title":"stepStatus",
+              "description":"Status of the journey step, representing in which shape the step is, when its processing has been done.",
+              "type":"string"
+            },
+            "https://ns.adobe.com/experience/journeyOrchestration/journeyId":{
+              "title":"journeyId",
+              "description":"ID of the journey.",
+              "type":"string"
+            },
+            "https://ns.adobe.com/experience/journeyOrchestration/journeyVersionId":{
+              "title":"journeyVersionId",
+              "description":"ID of the journey version. This id represents the identity reference to the journey, in case of the journeyStepEvent.",
+              "type":"string"
+            },
+            "https://ns.adobe.com/experience/journeyOrchestration/journeyVersionName":{
+              "title":"journeyVersionName",
+              "description":"Name of the journey version.",
+              "type":"string"
+            },
+            "https://ns.adobe.com/experience/journeyOrchestration/journeyVersion":{
+              "title":"journeyVersion",
+              "description":"Version of the journey version.",
+              "type":"string"
+            },
+            "https://ns.adobe.com/experience/journeyOrchestration/instanceId":{
+              "title":"instanceId",
+              "description":"Internal ID of the journey instance.",
+              "type":"string"
+            },
+            "https://ns.adobe.com/experience/journeyOrchestration/externalKey":{
+              "title":"externalKey",
+              "description":"External key extracted from the event to process it.",
+              "type":"string"
+            },
+            "https://ns.adobe.com/experience/journeyOrchestration/parentStepId":{
+              "title":"parentStepId",
+              "description":"Step ID of the parent of the current processed step in the instance.",
+              "type":"string"
+            },
+            "https://ns.adobe.com/experience/journeyOrchestration/parentStepName":{
+              "title":"parentStepName",
+              "description":"Step name of the parent of the current step.",
+              "type":"string"
+            },
+            "https://ns.adobe.com/experience/journeyOrchestration/parentTransitionId":{
```

Comment:
Since the schema is already in the denormalized state, with ID and NAME captured. Are there any other attributes which you need to lookup against this ID? Otherwise, these ID's are irrelevant.